feature-policy/resources/picture-in-picture.js

```diff
@@ -1,7 +1,18 @@
-function isPictureInPictureAllowed() {
-  if (!('pictureInPictureEnabled' in document))
-    return Promise.resolve(false);
+function async_pip_test(func, name, properties) {
```

Comment:
There aren't any properties that do anything useful, so I would drop that argument. (It's cleanup that nobody has gotten to.)